### PR TITLE
User feed

### PR DESCRIPTION
**Pull request by foolhardy21**
## User Feed Page.

This PR has the UI components for user feed page. The posts of user's following are displayed on this page.

- Files to review:
   - [ ]  src/pages/UserFeed.jsx
   - [ ] src/reducers/posts.reducer.js

- Use this link for preview - https://deploy-preview-5--remarkable-flan-74a807.netlify.app/login

v-1

Thanks

### CODE REVIEW

File: src/components/Reusable/reusable.module.css [hunk 1]
```diff
@@ -9,11 +9,24 @@
   height: 100vh;
 }
 
+.feedDiv {
+  grid-area: feed;
+  overflow-y: auto;
+  height: 100vh;
+  border-left: solid thin var(--clr-off-primary);
```
Comment (Dev-AkshatGupta) [inline src/components/Reusable/reusable.module.css:16]: Very first time saw property thin

File: src/pages/UserFeed.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                })
+            } catch (e) {
+                console.log(e.response)
+            }
+            // get mohit(loggedin) with updated following array
+            const response3 = await axios.get(`/api/users/${mohit[0]._id}`)
+            // extract the following of mohit(loggedin)
+            const mohitfollowing = response3.data.user.following.map(user => user.username)
+            // set the user feed with posts of mohit(loggedin)'s following
+            postsDispatch({ type: ACTION_INIT_USER_FEED, payload: mohitfollowing })
```
Comment (anik31) [inline src/pages/UserFeed.jsx:36]: appreciation - good that you're using comments

File: src/pages/UserFeed.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+const UserFeedSection = () => {
+    const { postsDispatch } = usePosts()
+    const { getUserToken } = useAuth()
+
+    // you may ignore this for now, this is a setup for the user feed
+    useEffect(() => {
+        (async () => {
+            const response1 = await axios.get('/api/users')
+            const mohit = response1.data.users.filter(user => user.username === 'coolmohit')
+            const peter = response1.data.users.filter(user => user.username === 'ranjanchauhan')
```
Comment (anik31) [inline src/pages/UserFeed.jsx:19]: suggestion - you can check the status code of the api call

**Review by anik31 — APPROVED** [review]
Everything looks good. Great work 💯

File: src/pages/UserFeed.jsx (hunk 3 above)
Comment (foolhardy21) [reply src/pages/UserFeed.jsx:19]: yes, this is only for now. ill remove this code once all the features are added.

**Pull request merged**